Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -1263,7 +1263,7 @@ internal ItemImageInfo ItemImageInfoFromValueString(ReadOnlySpan<char> value)
         /// <param name="id">The id.</param>
         /// <returns>BaseItem.</returns>
         /// <exception cref="ArgumentNullException"><paramref name="id"/> is <c>null</c>.</exception>
-        /// <exception cref="ArgumentException"><paramr name="id"/> is <seealso cref="Guid.Empty"/>.</exception>
+        /// <exception cref="ArgumentException"><paramref name="id"/> is <seealso cref="Guid.Empty"/>.</exception>
```

Comment:
I'm not certain about this, but there wasn't any strong hint justifying the divergence..